other/pod_lifetime_annotation/policy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-lifetime
+  annotations:
+    policies.kyverno.io/title: Enforce pod duration
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4,2
```

Comment:
comma between "4" and "2" rather than period

Reply:
Fixed the typo